### PR TITLE
Test 2

### PR DESCRIPTION
**Pull request by BuhtigithuB**
I try to make PR to see if merge have been done correctly... Hope this one pass Travis...

### CODE REVIEW

File: gluon/globals.py [hunk 1]
```diff
@@ -1207,9 +1205,3 @@ def _close(self, response):
                 del response.session_file
             except:
                 pass
-
```
Comment (niphlod) [inline gluon/globals.py]: ehm.....start from master, else CHECK before committing a file that DROPS a method....

**Comment by niphlod** [comment]
you keep screwing with submodule's tracking. That's the bit failing, not your pep8-ized work. 

Also, pleeeease squash those 30 commits into one. And again, maybe before pressing the "send PR" button, triple-check what you're sending in :-P

**Comment by BuhtigithuB** [comment]
Shit I am crying such a waste of time... It coming from pyDal?

**Comment by BuhtigithuB** [comment]
I clone my fork with like this : git clone  https://github.com/BuhtigithuB/web2py.git

Should I did something else for pyDal??

Thanks

**Comment by niphlod** [comment]
as explained in https://github.com/web2py/web2py/pull/836 , there's a whole section on the book on how to contribute (and fork, and managing submodules, and keeping it in sync with master, and enabling tests, and ensure a fast-forward merge, and squashing commits, and some other tricks)

**Pull request closed without merging**